frontend/src/components/adjudications/AdjudicationSection.jsx

```diff
@@ -5,18 +5,21 @@ import pick from 'lodash/pick';
 import AdjudicationTableRow from './AdjudicationTableRow';
 import Section from '../interface/Section';
 import { getPerformances } from '../../api/PerformanceAPI';
+import { getNextUnjudgedPerformance } from "../../api/AdjudicationAPI";
 import humps from 'humps';
 import { useState, useEffect } from 'react';
+import { Button } from '@material-ui/core';
 
 export default function AdjudicationsSection(props) {
 
   const [loading, setLoading] = useState(true)
   const [adjudications, setAdjudications] = useState({}) 
   const [performances, setPerformances] = useState({}) 
   const collectionName = `events/${eventId}/performances/1/adjudications`; //replace 1 with ${performanceId}
-  const [filteredPerformances, setFilteredPerformances] = useState([]) 
+  const [filteredPerformances, setFilteredPerformances] = useState([])
```

Comment:
nit: add `;`

Reply:
anything that isn't control flow (if/else), JSX, or within an object should end in `;`